src/nimble_mpi.cc

```diff
@@ -564,11 +525,7 @@ int ExplicitTimeIntegrator(
       displacement(i, 2) += delta_time * velocity(i, 2);
     }
 
-#ifdef NIMBLE_HAVE_UQ
-    // advance approximate trajectories
-    uq_model.UpdateDisplacement(delta_time);
```

Comment:
i.e. in the uq code this is just displacement+= delta_time * velocity; for all the auxillary trajectories